src/Faker/Provider/pt_BR/Person.php

```diff
@@ -99,4 +99,56 @@ public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+
+    /**
+     * A random CPF number.
+     * @param bool $formatted If the number should have dots/dashes or not.
+     * @return string
+     */
+    public function cpf($formatted = true)
+    {
+        $n = $this->generator->numerify('#########');
+        $n .= $this->verifierDigit($n);
```

Comment:
I do it that way because the first digit verifies the document number, and the second one verifies the document number and the first digit itself. Thus, for modularity, I've created a method that only calculates a digit, instead of generating all the needed digits. It could be used later for other types of documents if they use the same modulus 11 algorithm, such as driver's license or RG (yes, we brazilians do have two different document numbers for identity).